dictionaries/CallMap_81_delta.php

```diff
@@ -23,6 +23,7 @@
     'imagecreatefromavif' => ['false|GdImage', 'filename'=>'string'],
     'mysqli_fetch_column' => ['null|int|float|string|false', 'result'=>'mysqli_result', 'column='=>'int'],
     'mysqli_result::fetch_column' => ['null|int|float|string|false', 'column='=>'int'],
+    'CURLStringFile::__construct' => ['void', 'data'=>'string', 'postname'=>'string', 'mime'=>'string'],
```

Comment:
The CallMap syntax uses an "=" at the end of an argument name to convey that the argument is optional. You should add that on `mime`